index.html

```diff
@@ -39,6 +39,71 @@ <h2>I'am a Software Developer</h2>
 </div>
 </div>
 </div>
+<div class="grid-container">
+    <div class="grid-item1">
+<ul>
+ <li class="head">My Recent Works</li>
+ <li class="border"></li>   
+<li id="img7"></li>
+<li class="post">Multi-Post Stories</li>
+<li class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</li>
+</ul>
+<ul class="tag">
+<li id="tag1">css</li>
+<li id="tag2">html</li>
+  <li id="tag3">bootstrap</li>
+  <li id="tag4">Ruby</li>
+  <li><button class="btn" type="button">See Project</button></li>
+</ul>
+<div>
+  <ul>
+    <li class="img8"></li>
+    <li class="post2">Profesional Art Printing Data</li>
+<li class="support2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</li>
+</ul>
+<ul class="tagB">
+<li id="tag5">html</li>
+  <li id="tag6">bootstrap</li>
+  <li id="tag7">Ruby</li>
+  <li><button class="btn1" type="button">See Project</button></li>
+</ul>
+</div>
+      <div class="tab">
+     <ul>
+       <li class="about">About me</li>
+       <li class="support3">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</li>
+       <li><button class="btn3" type="button">Get My Resume</button></li>
+      <li class="border1"></li>
+      </ul>
+      <ul class="lang">
+        <li id="star"></li>
+        <li class="main">Language</li>
+        <li class="java">Javascript</li>
+        <li class="ruby">Ruby</li>
+        <li class="html">Html</li>
+        <li class="css">Css</li>
+      </ul>
+      <ul class="fw">
+        <li id="box"></li>
+        <li id="fw">Frameworks</li>
```

Comment:
- [ ] To keep the HTML semantic, you should consider removing the title and the respective icon outside this `<ul>`.
This is my structure suggestion: 

```
<ul>
  <li>Language</li>
    <ul>
     <li>JS</li>
     ...
    </ul>
  <li>Frameworks>
    <ul>
      <li>Ruby on rails</li>
      ...
    </ul>
  <li>Skills</li>
    <ul>
      <li>Codekit</li>
      ...
    </ul>
</ul>
```